components/global.scss

```diff
@@ -3,330 +3,350 @@
 /*************/
 
 // height and width h-0 through h-100, w-0 through w-100
-$dim-start:	0 !default;
-$dim-end:	100 !default;
-$interval:	5 !default;
-$dimensions: (height: h, width: w);
+$dim-start: 0 !default;
+$dim-end: 100 !default;
+$interval: 5 !default;
+$dimensions: (
+  height: h,
+  width: w,
+);
 
 @each $dimension, $dim in $dimensions {
-	$i: $dim-start;
-	
-	@while $i <= $dim-end {
-		.#{$dim}-#{$i} {
-			#{$dimension}: #{$i}#{'%'};
-				
-			$i: $i + $interval;
-		}
-	}
+  $i: $dim-start;
+
+  @while $i <= $dim-end {
+    .#{$dim}-#{$i} {
+      #{$dimension}: #{$i}#{'%'} !important;
```

Comment:
Why the need for `!important` (here and below)?

Reply:
often our classes are outspecified by material and I also noticed bootstrap global utils were important. Do you see a reason to not do this?

Reply:
From what I understand, using `!important` in your CSS is a code smell and often indicates a problem.

[This article](https://css-tricks.com/when-using-important-is-the-right-choice/) is an example of why... but utility classes (like those here) are something they say is a good use case for "!important", so I retract my objection. 🙂